prometheus/datadog_checks/prometheus/__init__.py

```diff
@@ -0,0 +1,11 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from . import prometheus
+
+PrometheusCheck = prometheus.PrometheusCheck
```

Comment:
same as above, `from .prometheus import PrometheusCheck` will spare the assignment